test/algorithms/single_histogram_threshold.jl

```diff
@@ -98,4 +98,17 @@
             @test img₀₁[minpos] == 0
         end
     end
+
+    @testset "Offset Arrays" begin
+        # Verify that the algorithms give equivalent results on OffsetArrays
+        img = imresize(testimage("cameraman"); ratio=0.25)
+        offset_img = OffsetArray(img, OffsetArrays.Origin(0))
+        for (fname, f) in threshold_methods
+            img₀₁ = binarize(img, f)
+            offset_img₀₁ = binarize(offset_img, f)
+            for i in zip(img₀₁, offset_img₀₁)
+                @test first(i) == last(i)
+            end
```

Comment:
maybe

```suggestion
            @test axes(offset_img₀₁) == axes(offset_img)
            @test img₀₁ == OffsetArrays.no_offset_view(offset_img₀₁)
```

Reply:
Great! Didn't know about `OffsetArrays.no_offset_view(offset_img₀₁)`!

Reply:
I removed the Appveyor webhook. I don't have Appveyor listed here either:  https://github.com/zygmuntszpak/ImageBinarization.jl/settings/installations